requirements.txt

```diff
@@ -12,7 +12,7 @@ Flask-OpenID==1.3.0
 feedgen==0.9.0
 feedparser==6.0.10
 pymacaroons==0.13.0
-requests==2.28.2
+requests==2.31.0
```

Comment:
I think this can actually be the cause of the other checks not passing. Can you try reverting it, see if it works?

Reply:
This was not was causing the action to fail, it was a potential fix mentioned in the original issue that I was trying. Not needed anymore though!